lib/scripts/text.dart

```diff
@@ -0,0 +1,24 @@
+class UpdateAppText {
+  static String ConvertNumber(String phoneNumber) {
+    String newPhoneNumber = '';
+    List list = phoneNumber.split('');
+    list.insert(4, ' (');
+    list.insert(7, ') ');
+    list.insert(11, '-');
+    list.insert(14, '-');
+
+    for (int i = 0; i < list.length; i++) {
+      newPhoneNumber += list[i];
+    }
+    return newPhoneNumber;
+  }
+
+  static String MakeTextCorrect(String text) {
```

Comment:
makeTextCorrect // bad name

What means correct? Provide more details in method name